apps/console/src/extensions/components/users/components/guests/onboarded-guest-user-list.tsx

```diff
@@ -678,7 +688,8 @@ export const OnboardedGuestUsersList: React.FunctionComponent<OnboardedGuestUser
                 } }
                 placeholders={ showPlaceholders() }
                 selectable={ selection }
-                showHeader={ true }
+                // Show headers only if saas feature status is true
```

Comment:
Unnecessary comment as the code is self-explanatory

Reply:
Removed with https://github.com/wso2/identity-apps/pull/4676/commits/c4f007b75af8af9f1a32f6705c326bf9e812a86c